pysite/views/api/bot/hiphopify.py

```diff
@@ -35,7 +35,7 @@
 
 class HiphopifyView(APIView, DBMixin):
     path = "/hiphopify"
-    name = "hiphopify"
+    name = "api.bot.hiphopify"
```

Comment:
Nope, just `bot.hiphopify` - although we aren't namespacing this correctly anyway, it should really be `/bot/hiphopify` if we're doing that